vuepress/docs/v7.0/docs/reference/deployment-structure.md

```diff
@@ -4,154 +4,128 @@ sidebarDepth: 2
 
 * GitHub: <https://github.com/entando/app-builder/>
 * DockerHub: <https://hub.docker.com/repository/docker/entando/app-builder/>
 
 #### Customization
-The _app-builder_ is customized as part of many Entando implementations. 
+The App Builder is customized as part of many Entando implementations. 
 It can be customized at runtime via micro frontends 
 [widget configuration](../../tutorials/create/mfe/widget-configuration.md).
 
-## component-manager(cm)
-The _component-manager_ provides the link between the entando-de-app (or your custom core instance) and the ECR. The _component-manager_ queries the entando-k8s service to fetch available 
-bundles that have been deployed as custom resources inside of an Entando cluster. 
-The _component-manager_ also manages the relationships between an Entando application and the 
-installed plugins. This can be seen in the plugin link custom resources in Kubernetes. 
+## Entando Component Manager (ECM)
+ECM provides the link between the **entando-de-app**, or your custom core instance, and the ECR. ECM queries the Entando Kubernetes service to fetch available 
+bundles that have been deployed as custom resources inside the cluster. 
+
+ECM also manages the relationships between an Entando application and the 
+installed plugins. This can be seen in the plugin link custom resource in Kubernetes. 
 
 * GitHub: <https://github.com/entando-k8s/entando-component-manager/>
 * DockerHub: <https://hub.docker.com/repository/docker/entando/entando-component-manager/>
 
-#### Customization
-It is unlikely that the _component-manager_ will be customized as part of an Entando implementation.
-
 ## entando-k8s-service
-The _entando-k8s-service_ acts as an abstraction layer to fetch data from kubernetes APIs. The primary 
-functionality is in discovering and making available for installation Entando plugins. The 
-_entando-k8s-service_ is invoked by the _component-manager_. 
+The **entando-k8s-service** acts as an abstraction layer to fetch data from Kubernetes APIs. Its primary 
+function is to discover and make available Entando plugins for installation. The 
+**entando-k8s-service** is invoked by the ECM. 
 * GitHub: <https://github.com/entando-k8s/entando-k8s-service/>
 * DockerHub: <https://hub.docker.com/repository/docker/entando/entando-k8s-service/>
 
-#### Customization
-It is very unlikely that the _entando-k8s-service_ will be customized as part of an Entando implementation.
-
-## keycloak
-The _entando-keycloak_ project is an extension of the base Keycloak images. The extension provides the default 
-themes for Entando, a customized realm and clients, and adds the Oracle ojdbc jars for connection to Oracle 
-databases.
+## Keycloak
+The **entando-keycloak** project is an extension of the base Keycloak images. It provides default themes for Entando, customized realm and clients, and Oracle JDBC jars for connecting to Oracle databases.
```

Comment:
"a customized realm" or "customized realms"

jars -> drivers. "jars" is a bit esoteric and a file type (should be JARs)